code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -191,6 +191,21 @@
 		new /obj/item/reagent_containers/glass/beaker( src )
 		new /obj/item/reagent_containers/glass/beaker( src )
 
+/obj/item/storage/box/iv_bags
+	name = "IV Bags"
+	desc = "A box full of empty IV bags."
+	icon_state = "beaker"
+
+	New()

```

Comment:
I realize the rest of the file is gross, but at least make this one fine in terms of absolute pathing.